basic-algorithms.js

```diff
@@ -1,7 +1,71 @@
 // Names and Input
+let hacker1 = "Cristina";
+console.log(`The driver's name is ${hacker1}`);
 
+let hacker2 = prompt("Which is the navigator's name?");
+console.log(`The navigator's name is ${hacker2}`);
 
 //Conditionals
-
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The Driver ${hacker1} has the longest name, it has ${
+      hacker1.length
+    } characters`
+  );
+} else if (hacker2.lenght > hacker1.length) {
+  console.log(
+    `Yo, navigator ${hacker2} got the longest name, it has ${
+      hacker2.length
+    } characters`
+  );
+} else if (hacker1.length === hacker2.length) {
+  console.log(
+    `wow, you both ${hacker1} and ${hacker2} got equally long names, ${
+      hacker1.length
+    } characters!!`
+  );
+}
 
 // Lorem ipsum generator
+let driver = hacker1;
+let navigator = hacker2;
+
+for (i = 0; i < driver.length; i++) {
+  console.log(driver[i].toUpperCase());
+  i += " ";
+}
+
+for (j = navigator.length -1; j >= 0; j--){
+  console.log(navigator[j]);
+  j += " ";
```

Comment:
Lo mismo que te comenté arriba, además que el ambito de las variables acaba cuando acaba el bucle, así que puedes hacer otro for utilizando la i en lugar de j.